### PR TITLE
US397426: Adding date to logs

### PR DESCRIPTION
**Pull request by tbroadbent22**
https://internal.almoctane.com/ui/?p=131002%2F6001&idpId=https%3A%2F%2Fauthenticate.microfocus.net%2Fnidp%2Fsaml2%2Fmetadata#/team-backlog/stories

### CODE REVIEW

**Comment by tbroadbent22** [comment]
Developer build: http://sou-jenkins2.swinfra.net:8080/job/CAFapi/job/CAFapi~caf-logging~US397426~CI

File: caf-logging-log4j2/src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 <Configuration>
     <Appenders>
-        <Console name="STDOUT" target="SYSTEM_OUT">
+        <Console name="STDOUT" target="SYSTEM_ERR">
```
Comment (dermot-hardy) [inline caf-logging-log4j2/src/main/resources/log4j2.xml]: Should the console name still be `STDOUT`?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
The Readme and the release notes haven't been updated.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
Same comment as last time:
The Readme and the release notes haven't been updated.

**Comment by tbroadbent22** [comment]
Apologies, didnt see the first comment regarding the README and Release Notes, both have now been updated

File: README.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ This project configures Logback and Log4j2 to meet the CAF Logging Standard.
     - Exception stack traces are JSON-encoded so that they do not span multiple lines.
 
 #### Pattern:
-    [(UTC Time) #(Process Id).(Thread Id) (Log Level) (Tenant Id) (Correlation Id)] (Logger): (Log Message)
+    [(Date) (UTC Time) #(Process Id).(Thread Id) (Log Level) (Tenant Id) (Correlation Id)] (Logger): (Log Message)
```
Comment (dermot-hardy) [inline README.md]:
```suggestion
    [(UTC Date) (UTC Time) #(Process Id).(Thread Id) (Log Level) (Tenant Id) (Correlation Id)] (Logger): (Log Message)
```

File: release-notes-1.4.0.md [hunk 3]
```diff
@@ -4,5 +4,6 @@
 ${version-number}
```
Comment (dermot-hardy) [inline release-notes-1.4.0.md]: - Release notes file needs to be renamed to align with the new version number.
- As this is a major version number there should be a "Breaking Changes" section.
Read the Release notes section in https://rdapps.swinfra.net/quixy/#/viewEntity/OCTFT52I355035.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: release-notes-2.0.0-SNAPSHOT.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+!not-ready-for-release!
```
Comment (dermot-hardy) [inline release-notes-2.0.0-SNAPSHOT.md]: - Remove the `-SNAPSHOT` from the version number in the release notes.  (I understand why you added it but it's not what the build process expects.)

File: release-notes-2.0.0-SNAPSHOT.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Version Number
+${version-number}
+
+#### New Features
+US397426: Adding date to logs and migrating to SYSTEM_ERR
+
+#### Known Issues
+
+#### Breaking Changes
```
Comment (dermot-hardy) [inline release-notes-2.0.0-SNAPSHOT.md]: When you're modifying Markdown files you need to view them to make sure the formatting is correct.
Look at how this is displaying:
https://github.com/CAFapi/caf-logging/blob/b697d2f23dac2b8d339362a8004d42905a02c654/release-notes-2.0.0-SNAPSHOT.md

The lines are running into each other.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: release-notes-2.0.0-SNAPSHOT.md (hunk 5 above)
Comment (tbroadbent22) [reply release-notes-2.0.0-SNAPSHOT.md]: Have now made these changes and will submit for re review

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**